include/stl2/detail/concepts/object/move_constructible.hpp

```diff
@@ -25,6 +25,10 @@ STL2_OPEN_NAMESPACE {
 	template <class>
 	constexpr bool __addressable = false;
 	template <class T>
+	constexpr bool __addressable<T&> = true;
+	template <class T>
+	constexpr bool __addressable<T&&> = true;
```

Comment:
This is right for `Destructible`, but wrong for `Addressable`: `Addressable` needs to be changed to `_Is<T, is_object> && __addressable<T>`. `Destructible` need not subsume `Addressable`, we can define it as:
```c++
namespace models {
  template <class T>
  constexpr bool Destructible = std::is_nothrow_destructible<T>::value && __addressable<T>;
}
template <class T>
concept bool Destructible() { return models::Destructible<T>; }
```